packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -97,18 +102,7 @@ export async function createNewFluidFile(
         url: odspUrl,
         headers: { [ClpCompliantAppHeader.isClpCompliantApp]: isClpCompliantApp },
     });
-    fileEntry.docId = odspResolvedUrl.hashedDocumentId;
-    fileEntry.resolvedUrl = odspResolvedUrl;
```

Comment:
Why are these 2 lines removed?

Reply:
As far as I understood, `fileEntry` was an input parameter to the `createNewFluidFile()` function and I wasn't convinced that altering of the input parameter was on purpose. But for the scope of this PR I can introduce these lines of code back, and maybe we can tackle this in a separate one.

Reply:
So, we don't have info on fileEntry before creating the file. This is an object and same object is passed to epochTracker where it is actually used. So, it needs to be updated once the file is created. Also, we can't expose any api on epochTracker to update these as that would mean anybody can change that. However, it is still ok to do it here because it is file creation flow, and we don't have info on file before it is created.